atlas-db-controller/databaseSchemaResource.go

```diff
@@ -231,7 +212,7 @@ func (c *Controller) updateDatabaseSchemaStatus(key string, schema *atlas.Databa
 
 	_, err := c.atlasclientset.AtlasdbV1alpha1().DatabaseSchemas(schema.Namespace).Update(schemaCopy)
 	if err != nil {
-		runtime.HandleError(fmt.Errorf("error updating status to '%s' for database schema '%s': %s", state, key, err))
+		logrus.Warningf("error updating status to '%s' for database schema '%s': %s", state, key, err)
```

Comment:
why runtime.HandleError is not used here?

Reply:
`runtime.HandleError()` is simply printing the error messages which we are doing anyway with logging statements. So removing them